Renderer/RenderProcess.php

```diff
@@ -100,7 +100,12 @@ private function render ()
      */
     private function renderTextSection (string $tagName, array $markers) : string
     {
-        return "<{$tagName}>" . $this->renderMarkers($markers) . "</{$tagName}>";
+        $matches = [];
+        $tagAttributes = 1 === \preg_match("#^h[1-6]$#", $tagName, $matches)
+            ? " class=\"{$matches[0]}\""
```

Comment:
Just remove the matches. As you have a full match, you can just do:

```php
$tagAttributes = 0 !== \preg_match("#^h[1-6]$#", $tagName)
    ? " class=\"{$tagName]}\""
    : "";
```

Reply:
You're absolutely right! Fixed!